src/eddymotion/model/base.py

```diff
@@ -127,6 +128,8 @@ def __init__(self, gtab, S0=None, mask=None, b_max=None, **kwargs):
         if not model_str:
             raise TypeError("No model defined")
 
+        # ToDo
+        # Use lazy loading ?
```

Comment:
Please open an issue to suggest incidental findings/actions
```suggestion
```

Reply:
Will do.